sdk/examples/wallet/output_consolidation.rs

```diff
@@ -3,84 +3,87 @@
 
 //! In this example we will consolidate basic outputs from an account with only an AddressUnlockCondition by sending
 //! them to the same address again.
-//! Rename `.env.example` to `.env` first.
 //!
-//! `cargo run --example output_consolidation --release`
+//! Make sure that `example.stronghold` and `example.walletdb` already exist by
+//! running the `create_account` example!
+//!
+//! Rename `.env.example` to `.env` first, then run the command:
+//! ```sh
+//! cargo run --release --all-features --example output_consolidation
+//! ```
+
+use std::env::var;
 
 use iota_sdk::{
-    client::{
-        constants::SHIMMER_COIN_TYPE,
-        secret::{mnemonic::MnemonicSecretManager, SecretManager},
-    },
     types::block::address::ToBech32Ext,
-    wallet::{ClientOptions, Result, Wallet},
+    wallet::{Result, Wallet},
 };
 
 #[tokio::main]
 async fn main() -> Result<()> {
     // This example uses secrets in environment variables for simplicity which should not be done in production.
     dotenvy::dotenv().ok();
 
-    let mnemonic: &str = &std::env::var("NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC_1").unwrap();
-    let mnemonic_secret_manager = MnemonicSecretManager::try_from_mnemonic(mnemonic).unwrap();
-    let secret_manager = SecretManager::Mnemonic(mnemonic_secret_manager);
-
-    let client_options = ClientOptions::new().with_node(&std::env::var("NODE_URL").unwrap())?;
-
-    // Create the wallet
     let wallet = Wallet::builder()
-        .with_secret_manager(secret_manager)
```

Comment:
Are these the default now?

Reply:
It's assumed that you run the `create_account` example before which is fine because a lot of other examples also have that one as a dependency. Then all that's needed is the storage path to build the wallet.

Reply:
I see.